src/Ledger/PParams.lagda

```diff
@@ -47,65 +46,60 @@ record PParams : Set where
 \emph{Network group}
 \AgdaTarget{maxBlockSize, maxTxSize, maxHeaderSize, maxValSize, pv}
 \begin{code}
-        maxBlockSize       : ℕ
-        maxTxSize          : ℕ
-        maxHeaderSize      : ℕ
-        maxValSize         : ℕ
-        pv                 : ProtVer -- retired, keep for now
+        maxBlockSize maxTxSize        : ℕ
```

Comment:
Put multiple fields of the same type on a single line to reduce the length of the record, but not too aggressively (i.e., don't make lines too long and don't change the ordering of fields).